helpers/JSON.go

```diff
@@ -5,18 +5,18 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io"
 	"io/ioutil"
 	"log"
 	"net/http"
 	"time"
+	"github.com/go-chi/render"
 
 	"google.golang.org/grpc"
 )
 
 func GetJsonFromPostRequest(r *http.Request, v interface{}) error {
 	if r.Method != "POST" {
-		return errors.New(fmt.Sprintf("expected POST request, received %s", r.Method))
+		return errors.New(fmt.Sprintf("Expected a POST request, received %s.", r.Method))
```

Comment:
Should be lowercase without period as by go standards
```suggestion
		return errors.New(fmt.Sprintf("expected a POST request, received %s", r.Method))
```

Reply:
See: https://github.com/golang/go/wiki/CodeReviewComments#error-strings